src/object/player.hpp

```diff
@@ -79,6 +79,7 @@ class Player final : public MovingObject,
   virtual void on_flip(float height) override;
   virtual bool is_saveable() const override { return false; }
   virtual bool is_singleton() const override { return false; }
+  virtual bool priority_in_list() const override { return true; }
```

Comment:
Doesn't say what kind of list. That shit is nondescript as fuck.

Reply:
may i recommend `has_object_manager_priority`

Reply:
You may. I'll allow it